deploy/helm/flightctl/values.yaml

```diff
@@ -13,29 +13,33 @@ global:
     # -- Key for the base domain wildcard certificate.
     key: ""
 # -- Storage class name for the PVCs.
-# storageClassName: standard
+  # storageClassName: standard
   # -- Name of the image pull secret for accessing private container registries
+  # You must create this secret manually before deployment
   imagePullSecretName: ""
-  # -- Alternative to global.auth.k8s.externalOpenShiftApiUrl with the same meaning, used by the multiclusterhub operator
-  apiUrl: "" # alternative to global.auth.k8s.externalOpenShiftApiUrl used by multiclusterhub operator
+  # -- External API URL for client access and authentication, used by the multiclusterhub operator
+  apiUrl: ""
   auth:
-    # -- Type of the auth to use. Can be one of 'k8s', 'oidc', 'builtin', 'aap', or 'none'
+    # -- Type of the auth to use. Can be one of 'k8s', 'oidc', 'builtin', 'aap', 'openshift', or 'none'
     # Note: 'builtin' is a legacy mode that translates to 'oidc' with PAM issuer automatically enabled
     # For new deployments, explicitly set type to 'oidc' and configure pamOidcIssuer settings
-    type: "k8s" # k8s, oidc, builtin (legacy), aap, none
+    # Leave empty (default) to auto-detect: 'openshift' on OpenShift clusters, 'k8s' otherwise
+    type: "" # Auto-detect (openshift on OpenShift, k8s otherwise), or explicitly set: k8s, oidc, builtin, aap, openshift, none
```

Comment:
can we get rid of `builtin` now ? Why keeping legacy option, when we dont have to ?
Also Im not entirely sure what the difference between auto and builtin option would be. It seems like unnecessary complication